StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1101CodeFixProvider.cs

```diff
@@ -67,5 +70,36 @@ private static Task<Document> GetTransformedDocumentAsync(Document document, Syn
 
             return Task.FromResult(document.WithSyntaxRoot(newSyntaxRoot));
         }
+
+        private class FixAll : DocumentBasedFixAllProvider
```

Comment:
The indentation for `Instance` and `FixAllInDocumentAsync { ... return ... }` is weird.